frontend/src/lib/components/leaderboard.svelte

```diff
@@ -19,15 +20,40 @@
         </Table.Row>
       </Table.Header>
       <Table.Body>
-        {#each data as { loses, name, wins, mmr }, idx}
+        {#each data as { loses, name, wins, mmr, mostRecentMMRChange, winningStreak, losingStreak }, idx}
           <Table.Row>
             <Table.Cell class="w-16 font-bold">{idx + 1}</Table.Cell>
             <Table.Cell>{name}</Table.Cell>
-            <Table.Cell>{wins}</Table.Cell>
-            <Table.Cell>{loses}</Table.Cell>
-            <Table.Cell class="text-right"
-              >{mmr != 0 ? mmr : '🐣'}</Table.Cell
-            >
+            <Table.Cell>
+              {wins}
+              {#if winningStreak && winningStreak >= SHOW_STREAK_THRESHOLD}
+                <span class="ml-2 text-xs" title="Winning streak"
+                  >🔥 {winningStreak}</span
+                >
+              {/if}
+            </Table.Cell>
+            <Table.Cell
+              >{loses}
+              {#if losingStreak && losingStreak >= SHOW_STREAK_THRESHOLD}
+                <span class="ml-2 text-xs" title="Losing streak"
+                  >🌧️ {losingStreak}</span
```

Comment:
Should it maybe just be the emoji and no actual count? Maybe the count is hover/tab on the emoji?

Reply:
Let's stick with this one in the beginning and try it out and see how people react to it